### PR TITLE
Closes #52: Add 404 page

### LINKED ISSUE #52: Create a 404 page

### PR DESCRIPTION
**Pull request by sharph**

### CODE REVIEW

File: app/404.html [hunk 1]
```diff
@@ -1,157 +1,113 @@
...
+    <section class="main-content">
+        <div class="container-fluid">
+            <div class="row">
+                <div class="col-xs-8 col-xs-push-2">
+                    <div class="content-title">
+                        <img class="content-title-icon" src="/images/museum-icon.svg">
+                        <h2>Page not found</h2>
+                        <hr>
+                    </div>
+                    <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse.</p>
```
Comment (CloudNiner) [inline app/404.html]: May want to get some actual 404 page text from @jbranigan or @designmatty before merging, so we don't push this and then forget about it.

**Comment by CloudNiner** [comment]
+1 consider adding the actual 404 page text now if available, otherwise, make issue so we don't forget.

File: app/404.html (hunk 1 above)
Comment (sharph) [inline app/404.html]: We'll want to update it anyway once there is footer text. How about we merge and create an issue?

Or just remove the paragraph under the `<h2>`?

Comment (CloudNiner) [inline app/404.html]: Sounds good. I'd remove the `<p>` and merge.

**Pull request merged**